README.rst

```diff
@@ -33,6 +33,17 @@ page. A `summary of contribution guidelines <CONTRIBUTING.md>`_ can also be
 used as a quick reference when you are ready to start writing or validating
 code for submission.
 
+Getting started with GitHub Codespaces
+----------------------
+
+To get started, create a codespace for this repository by clicking this 👇
```

Comment:
This came out of some discussion in an astropy dev telecon - we want to make it clear that this isn't a requirement or anything, but just a useful way to get started quickly.

```suggestion
Codespaces is a cloud development environment supported by GitHub.  None of the Astropy build machinery depends on it, but it is a convenient way to quickly get started doing development on Astropy.

To get started, create a codespace for this repository by clicking this 👇
```